assets/queries/openAPI/server_object_variable_not_used/query.rego

```diff
@@ -79,6 +79,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	server := doc.components.responses[r].links[l].server
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)